README.md

```diff
@@ -93,6 +109,7 @@ making TCP ipv4 `connect` syscalls like this one of me connecting to Freenode in
   "timestamp": "2019-11-12T14:24:57.532744Z",
   "pid": 1775,
   "daddr": "185.30.166.37",
+  "saddr": "X.X.X.X",
```

Comment:
nit: not entirely true. Won't it just pull the IP from /etc/hosts, which _may_ contain ipv6 address?

:meh:

Reply:
No it will never match v6 addresses because saddr only ever outputs v4 ones so they'll get ignored